good-enough-practices-for-scientific-computing.tex

```diff
@@ -258,13 +271,15 @@ \section{Data Management}\label{sec:data}
   create a supplementary file in order to look this up. Make sure to
   use the same names and codes when variables in two datasets refer to
   the same thing to simplify merging and lookup (and make them less
-  error-prone).  \fixme{Not sure what this point is asking for---the
-    term ``marshal'' is new to me in this sense.  Arjun Raj says,
+  error-prone).
+
+  \fixme{Not sure what this point is asking for---the
+    term ``marshal'' is new to me in this sense. Justin replies: Changed to include. Arjun Raj says,
     ``The use of the word ``standards'' is a bit confusing to me.  I
     think the point of this paragraph is to make sure that data is
     internally consistent, right?  And to add metadata?  The word
     ``standard'' here seems to imply something about quality, not
-    about consistency/labeling.''}
+    about consistency/labeling.'' Justin replies: Word standard no longer here.}
```

Comment:
Well, it still is part of the title of this recommendation

>Include complementary data in files that meet the same standards

Or did I misunderstand?